### PR TITLE
Create Docker build pipeline in Github action

### PR DESCRIPTION
**Pull request by 9to6**
I believe this repo need pipeline to build Docker images.
So I create GitHub action workflow and then, update Dockerfile.

After this PR is merged, need to create Release in the GitHub Release page for building Docker images.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -12,3 +12,4 @@ ADD . $SRC_DIR
 
 RUN cd $SRC_DIR/klayslave && go build -ldflags "-linkmode external -extldflags -static"
 RUN cp $SRC_DIR/klayslave/klayslave $PKG_DIR/bin
+RUN pip3 install locust==1.2.3
```
Comment (odg0318) [inline Dockerfile:15]: What do you think of refactoring Dockerfile like the following?

```dockerfile
RUN pip3 install locust==1.2.3

WORKDIR $SRC_DIR

ADD . .

RUN (cd klayslave && \
        go build -ldflags "-linkmode external -extldflags -static" && \
        cp klayslave $PKG_DIR/bin)
```

File: .github/workflows/workflow.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          platforms: linux/amd64,linux/arm64
+          push: ${{ github.event_name != 'pull_request' }}
```
Comment (odg0318) [inline .github/workflows/workflow.yml]: Why do we need this expression here?
```suggestion
          push: true
```

File: .github/workflows/workflow.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          platforms: linux/amd64,linux/arm64
```
Comment (odg0318) [inline .github/workflows/workflow.yml:35]: Looks very good!

Comment (9to6) [reply .github/workflows/workflow.yml:35]: @odg0318 I've tested it. But unfortunately it cannot be used it due to base image not supported arm64 yet. I will remove it.

**Review by odg0318 — APPROVED** [review]

File: Dockerfile [hunk 4]
```diff
@@ -1,14 +1,17 @@
...
-
-ENV PKG_DIR /locust-docker-pkg
 ENV SRC_DIR /go/src/github.com/klaytn/klaytn-load-tester
 ENV GOPATH /go
 
-RUN mkdir -p $PKG_DIR/bin
+WORKDIR $SRC_DIR
+ADD . .
+
+RUN (cd klayslave && \
```
Comment (yoomee1313) [inline Dockerfile]: Hi @9to6, we have `make build` to build the klayslave binary. Can't we use make command in docker? Sorry, i'm not familar with docker so if this question makes no sense, please tell me.

Comment (9to6) [reply Dockerfile]: @yoomee1313 I've updated it.

**Review by yoomee1313 — APPROVED** [review]

**Pull request merged**